### ISSUE TITLE
Custom soundfont player

### ISSUE THREAD

**Issue by jimutt** [jimutt/osmd-audio-player#10]
I'd like to avoid the current dependency of soundfont-player and supply a custom web audio pipeline for playback to gain more control of the low level audio features.

**Comment by ilionic** [comment]
Interesting, potentially scheduler could be simplified if audio would be running on separate thread ( based on AudioWorklet )

**Comment by jimutt** [comment]
Yes, I haven't really done enough reading on AudioWorklet yet to know the scope of use in detail and how it would be best used in a case like this. But it's absolutely something to look into more. And I saw that there there's support for it in Firefox 76 now as well.
